File: app/src/main/java/me/imzack/app/end/view/contract/ReminderViewContract.java
package me.imzack.app.end.view.contract;

public interface ReminderViewContract extends BaseViewContract {

    void showInitialView(String content, boolean hasDeadline, String deadline);

    void playEnterAnimation();

    void showToast(String msg);

    void enterPlanDetail(int position);
}

File: app/src/main/kotlin/me/imzack/app/end/view/contract/ReminderViewContract.kt
package me.imzack.app.end.view.contract

interface ReminderViewContract : BaseViewContract {

    fun showInitialView(content: String, hasDeadline: Boolean, deadline: String?)

    fun playEnterAnimation()

    fun showToast(msg: String)

    fun enterPlanDetail(position: Int)
}
